src/Media.js

```diff
@@ -53,16 +53,15 @@ const Media = (props) => {
   const classes = mapToCssModules(classNames(
     className,
     {
-      'media-body': body,
+      'flex-grow-1': body,
       'media-heading': heading,
```

Comment:
So the only classNames bootstrap had were `media` and `media-body`. Not sure what the other class names were used for. So I'll remove them if there is no use of them in our use case as I would assume they are not really used elsewhere either.

Reply:
Okay cool.  Why does this PR leave media-heading, media-list, & media-object though?

Reply:
Actually I only left it as a discussion point. Was going to remove it once not needed, so let me update that real quick.